### PR TITLE
Fixed crash, when mHost (hosting Activity) is null for dialog fragment.

### PR DESCRIPTION
**Pull request by orcchg**

### CODE REVIEW

File: frag-nav/src/main/java/com/ncapdevi/fragnav/FragNavController.kt [hunk 1]
```diff
@@ -471,8 +471,12 @@ class FragNavController internal constructor(builder: Builder, savedInstanceStat
             currentDialogFrag.dismiss()
             mCurrentDialogFrag = null
         } else {
-            val fragmentManager: FragmentManager = this.currentFrag?.childFragmentManager
-                    ?: this.fragmentManger
+            val fragmentManager: FragmentManager =
+                    if (this.currentFrag != null && this.currentFrag!!.isAdded) {
```
Comment (mateherber) [inline frag-nav/src/main/java/com/ncapdevi/fragnav/FragNavController.kt]: Please extract this.currentFrag to a local variable then you'll not need to use !!

Comment (orcchg) [reply frag-nav/src/main/java/com/ncapdevi/fragnav/FragNavController.kt]: done in next commit

**Comment by mateherber** [comment]
LGTM

**Comment by ncapdevi** [comment]
@orcchg  Yup, looks great, thanks for the PR!

**Pull request merged**